src/components/Dropdown/Dropdown.tsx

```diff
@@ -74,6 +81,7 @@ export function Dropdown(props: PropsWithChildren<DropdownProps>): JSX.Element {
   }, [parentQuery, lastTypedOrSubmittedValue, updateFocusedItem, setLastTypedOrSubmittedValue]);
 
   useRootClose(containerRef, () => {
+    onBlur?.(lastTypedOrSubmittedValue, value, focusedIndex, focusedItemData);
```

Comment:
Nitting but should onBlur also run when you use the tab key to leave the Dropdown? Maybe we could update FilterSearch to use `onToggle` instead?